protocol/metrics/metrics_consumer_manager.go

```diff
@@ -28,14 +28,16 @@ type ConsumerMetricsManager struct {
 	LatestBlockMetric                      *prometheus.GaugeVec
 	LatestProviderRelay                    *prometheus.GaugeVec
 	virtualEpochMetric                     *prometheus.GaugeVec
+	apiSpecificsMetric                     *prometheus.GaugeVec
 	endpointsHealthChecksOkMetric          prometheus.Gauge
 	endpointsHealthChecksOk                uint64
 	lock                                   sync.Mutex
 	protocolVersionMetric                  *prometheus.GaugeVec
 	providerRelays                         map[string]uint64
+	addMethodsApiGauge                     bool
 }
 
-func NewConsumerMetricsManager(networkAddress string) *ConsumerMetricsManager {
+func NewConsumerMetricsManager(networkAddress string, addMethodsApiGauge bool) *ConsumerMetricsManager {
```

Comment:
I don't like that this option is transferred like this. It should either be a global flag, being set from Cobra, or an options struct, with flags like this one, which will be probably be added in the future